.editorconfig

```diff
@@ -0,0 +1,9 @@
+# http://EditorConfig.org
+root = true
+
+[*.js]
+charset = utf-8
+indent_style = tab
+indent_size = 2
```

Comment:
I think `indent_size` is not needed with `indent_style = tab` but am not 100% sure. @karfau?

Reply:
It does have the effect of setting the size when previewed on GitHub, at the very least.

Reply:
It stays then, thanks.